IM_calculation/Advanced_IM/advanced_im_config.yaml

```diff
@@ -46,4 +46,6 @@ Haselton_RC_20Story:
 SDOF_IMK:
   script_location: 'Models/SDOF_IMK/run.py'
   timeout_threshold: null
-  
\ No newline at end of file
+SDOF_IMK_RotD:
```

Comment:
Storing a component in the IM name seems like a different style to how the rest of the IMs work. I'll need a deeper dive into this code - but I'm curious to know why you chose this architecture.

What's the difference in the TCL between SDOF_IMK and SDOF_IMK_RotD?

Reply:
The TCL itself handles the rotd logic (projecting the timeseries into the rotd directions) instead of doing it via python. Hence my question as to why we're doing it via TCL instead of via python like for simple IMs.